text/0002-keyboard-accessibility.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: March 16, 2023
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+
+We need a keyboard accessibility standard for Webstudio, since global standards only provide recommendations in many cases, but we need to be consistent.
+
+# Detailed design
+
+## Which elements should be focusable?
+
+All interactive elements should be focusable by keyboard. Interactive elements are those that do anything user-facing on mouse or keyboard interactions, like click or enter.
+
+Example: A button or a link should always be focusable, whereas a div that has no interactive usage should never be focusable.
+
+## Vertical lists of focusable items
+
+Long lists with focusable items are hard to exit if we tab through items. This is why once the first list element is focused, we choose to use arrow keys to navigate focus in the list.
```

Comment:
> Long lists with focusable items are hard to exit

These better call "tabbable" for clarity. All are focusable, tabbable are focusable which can be navigated with tabs.

Reply:
On the fresh look, there can be a list where each list item is not focusable, but we still want to be able to use arrow keys to navigate them instead of tabs

Reply:
actually I take it back, I checked https://react-spectrum.adobe.com/react-aria/useGridList.html
each list item there is focusable, even though it has no action attached

Reply:
added an example of a list item as a focusable item without action attached